LoopKit/DeviceManager/CGMManager.swift

```diff
@@ -94,6 +95,8 @@ public protocol CGMManager: DeviceManager {
     var cgmManagerStatus: CGMManagerStatus { get }
 
     /// Performs a manual fetch of glucose data from the device, if necessary
+    /// If no new data is avaliable to report and the last data reported was .unrealiableData,
```

Comment:
[nit 1] typos: `avaliable` -> `available`, `.unrealiableData` -> `.unreliableData`
[nit 2] may I suggest being a bit more prescriptive?  How about:

```
/// Implementations of this function must call the `completion` block, with the appropriate `CGMReadingResult`
/// according to the current available data.
/// - If the current available data is unreliable, return `.unreliableData`
/// - If the current available data is reliable, return `.newData` with the data samples
/// - Otherwise, if there is no current available data, return `.noData`
/// - If there is an error, return `.error` with the appropriate error.
```

Reply:
I take your suggest of being more prescriptive, but I used the following:

```
/// Implementations of this function must call the `completion` block, with the appropriate `CGMReadingResult`
/// according to the current available data.
/// - If there is new unreliable data, return `.unreliableData`
/// - If there is no new data and the current data is unreliable, return `.unreliableData`
/// - If there is new reliable data, return `.newData` with the data samples
/// - If there is no new data and the current data is reliable, return `.noData`
/// - If there is an error, return `.error` with the appropriate error.
```